manager/controllers/default/security/login.class.php

```diff
@@ -253,12 +259,11 @@ public function checkForAllowManagerForgotPassword() {
      *
      * @return void
      */
-    public function handleForgotLoginHash()
-    {
-        // Legacy workaround
-        if (!empty($_GET['modahsh'])) {
-            $_GET['modhash'] = $_GET['modahsh'];
-        }
+    public function handleForgotLoginHash() {
+        // Legacy workaround || wuuti: is that still necessary???
+//        if (!empty($_GET['modahsh'])) {
```

Comment:
We have git to be able of restoring things, if they turn out to be necessary. Either remove it if you believe it to no longer be necessary, or leave it if you think it is. If you're not sure if it is, then discuss it with people elsewhere (not in the code) and then incorporate the result.